tools/accuracy_checker/accuracy_checker/adapters/attribute_classification.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2019 Intel Corporation
```

Comment:
please apply correct license years for all modified files
```suggestion
Copyright (c) 2019-2020 Intel Corporation
```